community/kernel/src/test/java/org/neo4j/kernel/builtinprocs/BuiltInProceduresTest.java

```diff
@@ -143,6 +145,7 @@ public void shouldListCorrectBuiltinProcedures() throws Throwable
             record( "db.labels", "db.labels() :: (label :: STRING?)" ),
             record( "db.propertyKeys", "db.propertyKeys() :: (propertyKey :: STRING?)" ),
             record( "db.relationshipTypes", "db.relationshipTypes() :: (relationshipType :: STRING?)" ),
+            record( "sys.auth.changePassword", "sys.auth.changePassword(password :: STRING?) :: ()" ),
```

Comment:
Too early to sub-namespace these. Let's stick with `sys.changePassword` for now and add more levels when we have significantly more built-ins. The presence of the word "password" already suggests this is auth-related.